_issues/clinton_foundation_crooked.md

```diff
@@ -6,7 +6,7 @@ categories: trust
   
 **Q: Isn't The Clinton Foundation corrupt?**  
   
-**A**: Nothing could be further from the truth. The Clinton Foundation is a world-renowned charitable organization, and has recieved top marks from independent charity watchdog groups. The only reason it is being attacked now is because Hillary Clinton is running for President.
+**A**: Nothing could be further from the truth. The Clinton Foundation is a world-renowned charitable organization, and has recieved top marks from independent charity watchdog groups. The only reason it is being attacked now is because Hillary Clinton is running for President. In fact, Indiana Govenor Mike Pence claimed in the recent Vice Presidential debate that the Clinton Foundation gave less than 10% to “charitable organizations.” The truth is the foundation spent about 87% on charitable work, earning the Clinton Foundation an "A" rating from [CharityWatch.org](https://www.charitywatch.org/ratings-and-metrics/bill-hillary-chelsea-clinton-foundation/478).
```

Comment:
Yeah. This spelling mistake was fixed in a separate commit. This PR is now out-of-date and needs to be rebased off the HEAD of master.